kafka-ui-react-app/src/components/common/NewTable/Table.tsx

```diff
@@ -244,6 +270,8 @@ const Table: React.FC<TableProps<any>> = ({
                 <S.Row
                   expanded={row.getIsExpanded()}
                   onClick={handleRowClick(row)}
+                  onMouseOver={handleRowHover(row)}
```

Comment:
do we need to actually to bind this event handlers all the time ?

Reply:
Yes, it is necessary for both of these handlers. Otherwise, they are not triggering at all

Reply:
what i mean is we can bind it conditionally , cause this way it always has a event handler but in the case where we don't pass we still have an event handler that we don't do anything with.

`onMouseOver={onRowHover ? handleRow : undefined}`

Reply:
Oh, i got it. Update mouse over/leave binds